src/hwc/integration/deploy_an_hwc_app_test.go

```diff
@@ -71,6 +71,47 @@ var _ = Describe("CF HWC Buildpack", func() {
 			})
 		})
 
+		Context("with an extension buildpack", func() {
+			BeforeEach(func() {
+				if !ApiHasMultiBuildpack() {
+					Skip("API does not have multi buildpack support")
+				}
+
+				// re-use the suite-built hwc_buildpack as an extension buildpack
+				// this prevents having to fixture an entire buildpack to test
+				otherHwcBuildpackFile := packagedBuildpack
```

Comment:
We wanted to get thoughts on this integration strategy. Basically, we use the same buildpack artifact as it's own supply to at least minimally prove that it can be used as a supply and consume a supply; plus avoid introducing a heavy-weight extension fixture and avoid relying on another system buildpack that may or may not be on the foundation.